File: backend/app/infrastructure/models/menu.go
package models

import (
	"github.com/uptrace/bun"
)

// MenuModel メニューのインフラストラクチャーモデル
type MenuModel struct {
	bun.BaseModel `bun:"table:menus"`

	MenuID      string  `bun:"menu_id,pk"`
	Name        string  `bun:"name,notnull"`
	Description string  `bun:"description"`
	Price       float64 `bun:"price,notnull"`
	ImageURL    string  `bun:"image_url"`
	IsSoldOut   bool    `bun:"is_sold_out,notnull,default:false"`
	CategoryID  string  `bun:"category_id,notnull"`
}

File: frontend/apps/admin/app/(authenticated)/bill/types/menu.ts
/** メニューアイテム */
export type Menu = {
  menu_id: string;
  name: string;
  price: number;
  category_id: string;
  image_url: string;
  description: string;
};

/** GET /api/menus レスポンス */
export type GetMenusResponse = {
  menus: Menu[];
  count: number;
};
